openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java

```diff
@@ -401,11 +404,29 @@ private LineString constructFreeHandRouteGeometry(GHRequest request) {
         return new GeometryFactory().createLineString(coords);
     }
 
+    public void matchTraffic() {
+        // Do the graph extension post processing
+        // Reserved for processes that need a fully initiated graph e.g. for match making
+        if (getGraphHopperStorage() != null && processContext != null && processContext.getStorageBuilders() != null) {
+            for (GraphStorageBuilder graphStorageBuilder : processContext.getStorageBuilders()) {
+                if (graphStorageBuilder instanceof HereTrafficGraphStorageBuilder) {
+                    try {
+                        ((HereTrafficGraphStorageBuilder) graphStorageBuilder).postProcess(this);
+                    } catch (SchemaException e) {
+                        LOGGER.error("Error building the here traffic storage.");
+                        throw new RuntimeException(e);
+                    }
+                }
+            }
+        }
+    }
+
     /**
 	 * Does the preparation and creates the location index
 	 */
 	@Override
 	protected void postProcessingHook(boolean closeEarly) {
+		matchTraffic(); // TODO: Must this be moved into GraphHopper like it was originally?
```

Comment:
Re the TODO:  depending on whether traffic data is to be used with TD ALT, it might actually be necessary to have the data available before creating the landmarks. This would be in particular true when traffic speed could override the default edge-speed by setting it to a higher value. In such scenario the maximum speed value needs to be considered for calculating LMs in order to ensure the correctness of the approximator.

Reply:
So, is there anything to do for now?

Reply:
Moved matchTraffic into graphhopper